operations/mimir-mixin/dashboards/ruler-reads-resources.libsonnet

```diff
@@ -0,0 +1,53 @@
+local utils = import 'mixin-utils/utils.libsonnet';
+local filename = 'mimir-reads-resources.json';
+
+(import 'dashboard-utils.libsonnet') {
+  [filename]:
+    ($.dashboard('Ruler reads resources') + { uid: std.md5(filename) })
```

Comment:
Should be "Remote ruler reads resources" given applies only to ruler in remote mode?